WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingHomeListItemViewHolder.kt

```diff
@@ -3,30 +3,88 @@ package org.wordpress.android.ui.posts
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Button
+import android.widget.ImageView
 import android.widget.TextView
-import androidx.recyclerview.widget.RecyclerView.ViewHolder
+import androidx.annotation.LayoutRes
+import androidx.recyclerview.widget.RecyclerView
 import org.wordpress.android.R
-import org.wordpress.android.ui.posts.PrepublishingHomeItemUiState.PrepublishingHomeUiState
+import org.wordpress.android.ui.posts.PrepublishingHomeItemUiState.PublishButtonUiState
+import org.wordpress.android.ui.posts.PrepublishingHomeItemUiState.HeaderUiState
+import org.wordpress.android.ui.posts.PrepublishingHomeItemUiState.HomeUiState
 import org.wordpress.android.ui.utils.UiHelpers
+import org.wordpress.android.util.DisplayUtils
+import org.wordpress.android.util.image.ImageManager
+import org.wordpress.android.util.image.ImageType.BLAVATAR
 
-class PrepublishingHomeListItemViewHolder(internal val parent: ViewGroup, val uiHelpers: UiHelpers) : ViewHolder(
-        LayoutInflater.from(parent.context).inflate(
-                R.layout.prepublishing_action_list_item, parent, false
-        )
-) {
-    private val actionType: TextView = itemView.findViewById(R.id.action_type)
-    private val actionResult: TextView = itemView.findViewById(R.id.action_result)
-    private val actionLayout: View = itemView.findViewById(R.id.action_layout)
-
-    fun bind(uiState: PrepublishingHomeItemUiState) {
-        uiState as PrepublishingHomeUiState
-
-        actionType.text = uiHelpers.getTextOfUiString(itemView.context, uiState.actionType.textRes)
-        uiState.actionResult?.let { resultText ->
-            actionResult.text = uiHelpers.getTextOfUiString(itemView.context, resultText)
+private const val ROUNDING_RADIUS_DP = 6
+
+sealed class PrepublishingHomeViewHolder(internal val parent: ViewGroup, @LayoutRes layout: Int) :
+        RecyclerView.ViewHolder(
```

Comment:
NitPick: The class declarations in this file have inconsistent spacing/line-breaks. Could you please make them consistent - ideally put them on one/two lines (they should all fit on max two lines with the 120 chars line limit)

Reply:
Okay, thanks! Is there any way we can modify the auto-formatting to do this? I will take note of this now though but sometimes when working, I just autoformat and move on.

Reply:
Thanks! I had missed this 🙏